src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
```

Comment:
The class IsoscelesTrapezoid should not begin with an empty line. Please remove the empty line before the class declaration.